recipes-kernel/linux/linux-raspberrypi_5.10.bb

```diff
@@ -1,4 +1,11 @@
+<<<<<<< Updated upstream
 LINUX_VERSION ?= "5.10.63"
+=======
+# SPDX-FileCopyrightText: 2012 Andrei Gherzan <andrei@gherzan.ro>
+# SPDX-License-Identifier: MIT
+
+LINUX_VERSION ?= "5.10.52"
+>>>>>>> Stashed changes
```

Comment:
this is invalid hunk, perhaps some merge conflict. it should be deleted.

Reply:
Same here...

Reply:
Let's fix this.